pal_event_server/lib/src/allowlist_default_rules.dart

```diff
@@ -20,7 +20,10 @@ final chatRegex = RegExp(r'\bchat\b', caseSensitive: false);
 final meetRegex = RegExp(r'\bmeet\b', caseSensitive: false);
 final mailRegex = RegExp(r'\bmail\b', caseSensitive: false);
 final calendarRegex = RegExp(r'\bcalendar\b', caseSensitive: false);
+final slidesRegex = RegExp(r'\bslides\b', caseSensitive: false);
```

Comment:
Do we need to make them more specific, e.g. `Google Slides`, `Google Sheets`, etc.  to avoid overcounting. Or is it better this way?

Reply:
I was trying to capture all instances of mail or calendar originally and followed that same pattern. With Slides and Sheets, perhaps we would be more likely to capture other things we did not want rather than other apps in the same category. I revised it.

Reply:
Thanks!